src/commonStyles/index.ts

```diff
@@ -0,0 +1,16 @@
+import { createGlobalStyle } from 'styled-components';
+import { normalize } from 'styled-normalize';
```

Comment:
https://necolas.github.io/normalize.css/8.0.1/normalize.css
do not install unnesessary deps.

Reply:
added file and uninstalled dep